mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/linkingRulesTest.feature

```diff
@@ -14,3 +15,37 @@ Feature: linking-rules tests
     And assert karate.sizeOf(response[0].authoritySubfields) > 0
     And assert response[0].bibField != null
     And assert response[0].authorityField != null
+    And assert response[0].autoLinkingEnabled != null
+
+  @Positive
+  Scenario: Get instance to authority rules by id - Should return appropriate rule
+    Given path '/linking-rules/instance-authority/' + defauldLinkId
+    When method GET
```

Comment:
Please add negative scenario for 404 status

Reply:
Ok